landlab/components/stream_power/cfuncs.pyx

```diff
@@ -4,6 +4,9 @@ cimport cython
 
 #from libc.math cimport fabs
 
+# as best as KRB can tell, the function _brentq is in c.
+# not sure what, if anything I need to do to 
+from scipy.optimize._zeros import _brentq as brentq
 
```

Comment:
ETA: I'm clearly behind the times on this. I think the direct import of _brentq that you already do is, as you note in the code, the optimum you can achieve. So we're looking good I think.